### PR TITLE
Fix/ola progress bar & ola kpi

### PR DESCRIPTION
**Pull request by griselmatosm**
Main changes:

- Ola Kpi (changed styles and added variant attribute)
- Ola Progress Bar (added disabled attribute)
- Ola Item List (added background-color attribute and changed some styles)

### CODE REVIEW

File: src/OlaItemList.js [hunk 1]
```diff
@@ -5,16 +5,32 @@ const tokens = require('./tokens');
...
     };
 
+    headStyle() {
+        return `
+    .ola_item-list {
+        margin-bottom:${tokens('size-7')};
+        width:100%;
+        border-radius: ${tokens('radius')};
+        border-collapse: separate;
+        border-spacing: ${tokens('size-4')};
```
Comment (oscarotero) [inline src/OlaItemList.js:22]: I think this add too much space around (thinking in mobile). Maybe `size-3` is enough.

File: src/OlaKpi.js [hunk 2]
```diff
@@ -3,56 +3,56 @@ const { BodyComponent } = require('mjml-core');
...
+        description: '',
+        variant: 'gray'
     }
-    .ola_kpi-description {
-        margin: ${tokens('size-1')} 0 0;
+
+    headStyle() {
+        return `
+    .ola_kpi-value span {
+            font-size: ${tokens('size-7')};
```
Comment (oscarotero) [inline src/OlaKpi.js:21]: This is a Headline style, so I recomend to use the tokens from `font-headline`. Example:

```js
font-size: ${tokens('font-headline', 'font-size')};
font-family: ${tokens('font-headline', 'font-family')}; 
line-height: ${tokens('font-headline', 'line-height')};
```

**Review by oscarotero — CHANGES_REQUESTED** [review]

File: src/OlaItemList.js (hunk 1 above)
Comment (griselmatosm) [reply src/OlaItemList.js:22]: perfect 🙂

File: src/OlaKpi.js (hunk 2 above)
Comment (griselmatosm) [reply src/OlaKpi.js:21]: Ok 👌

**Review by oscarotero — APPROVED** [review]

**Pull request merged**